etna/analysis/plotters.py

```diff
@@ -1035,3 +1036,94 @@ def plot_imputation(
 
         ax[i].set_title(segment)
         ax[i].tick_params("x", rotation=45)
+
+
+def plot_periodogram(
+    ts: "TSDataset",
+    period: float,
+    amplitude_aggregation_mode: Union[str, Literal["per-segment"]] = AggregationMode.mean,
+    periodogram_params: Optional[Dict[str, Any]] = None,
+    segments: Optional[List[str]] = None,
+    columns_num: int = 2,
+    figsize: Tuple[int, int] = (10, 5),
+):
+    """Plot the periodogram to determine the optimal order parameter for `etna.transforms.FourierTransform`.
+
+    Parameters
+    ----------
+    ts:
+        TSDataset with timeseries data
+    period:
+        the period of the seasonality to capture in frequency units of time series, it should be >= 2
```

Comment:
Isnt it too difficult?

Reply:
What is the alternative? We have done this parameter like in FourierTransform.